### DISCUSSION TITLE
operation_commonについて

### DISCUSSION THREAD

**Discussion by sankakumusic** [kkksb/python_bank#6] [Ideas]
pythonだとワンライナーでこんなふうに書けます。` {i+1:e for i,e in enumerate(("show", "deposit", "withdraw", "exit"))}`
クラスというより定数にしてインポートしたほうが解釈が限定的(コンテキストが明確)になるので分かりやすくなりそうです。

```
OPERATION_MAP =  {i+1:e for i,e in enumerate(("show", "deposit", "withdraw", "exit"))}
```
https://github.com/kkksb/python_bank/blob/main/pybank/libs/operation_common.py

**Comment by kkksb** [comment]
@sankakumusic 
辞書型でも内包表現(?)できたんですね😀
モジュールに定数としておくかは、ちょっと検討してみます。
←定数もクラスでおいてあげると、ほかの定数を作成したときに分類するのが楽かなーと思ってました。(オブジェクト指向っぽい)
